pkg/security/utils/proc.go

```diff
@@ -64,32 +64,43 @@ func GetProcessNetworkNamespace(nsPath string) (uint32, error) {
 
 // CgroupTaskPath returns the path to the cgroup file of a pid in /proc
 func CgroupTaskPath(tgid, pid uint32) string {
-	return filepath.Join(util.HostProc(), fmt.Sprintf("%d/task/%d/cgroup", tgid, pid))
+	return pidPath(uint64(tgid), "task", strconv.Itoa(int(pid)), "cgroup")
+}
+
+func pidPath(pid uint64, combineWith ...string) string {
+	p := []byte(util.HostProc)
```

Comment:
why not returning
filepath.Join(append([]string{util.HostProc(),strconv.FormatUint(pid, 10)}, combineWith...)...)

Reply:
`strconv.FormatUint(pid, 10)` will allocate